native/kotlin/example/composeApp/src/commonMain/kotlin/rs/wordpress/example/shared/ui/login/LoginScreen.kt

```diff
@@ -24,7 +24,7 @@ fun LoginScreen(authenticateSite: (String) -> Unit) {
             verticalArrangement = Arrangement.Center,
             modifier = Modifier.fillMaxSize(),
         ) {
-            var siteUrl by remember { mutableStateOf("magnificent-funnel.jurassic.ninja") }
+            var siteUrl by remember { mutableStateOf("vanilla.wpmt.co") }
```

Comment:
Addresses [this suggestion.](https://github.com/Automattic/wordpress-rs/pull/720#discussion_r2093725697)